### PR TITLE
updated fixes in nostr and gitea vue

### PR DESCRIPTION
**Pull request by mik-tf**
# Work Done

- Small fix in the Nostr and Gitea vue

### CODE REVIEW

**Review by zaelgohary — DISMISSED** [review]

**Review by maayarosama — APPROVED** [review]

**Comment by mik-tf** [comment]
EDIT: I fixed some other typos.

File: packages/playground/src/weblets/tf_kubernetes.vue [hunk 1]
```diff
@@ -131,7 +131,7 @@ async function deploy() {
     layout.value.setStatus("success", "Successfully deployed a Kubernetes cluster.");
     layout.value.openDialog(k8s, deploymentListEnvironments.k8s);
   } catch (e) {
-    layout.value.setStatus("failed", normalizeError(e, "Failed to deploy kubernetes cluster."));
+    layout.value.setStatus("failed", normalizeError(e, "Failed to deploy Kubernetes cluster."));
```
Comment (zaelgohary) [inline packages/playground/src/weblets/tf_kubernetes.vue]: a Kubernetes cluster.

**Review by zaelgohary — CHANGES_REQUESTED** [review]

File: packages/playground/src/weblets/tf_kubernetes.vue (hunk 1 above)
Comment (mik-tf) [reply packages/playground/src/weblets/tf_kubernetes.vue]: Yes! Was fixed in the 2nd commit. thanks and good catch!

Comment (zaelgohary) [reply packages/playground/src/weblets/tf_kubernetes.vue]: It's not fixed yet.

Comment (mik-tf) [reply packages/playground/src/weblets/tf_kubernetes.vue]: Oh my bad, I thought you meant to change k to K. But the "a" was missing. Just committed the change. Thanks!

**Review by zaelgohary — APPROVED** [review]
LGTM

**Pull request merged**